packages/js/src/notifications/helpers.ts

```diff
@@ -12,7 +13,7 @@ export const read = async ({
   emitter: NovuEventEmitter;
   apiService: InboxService;
   args: ReadArgs;
-}): Promise<Notification> => {
+}): Result<Notification> => {
```

Comment:
The `Result` is the type that is promise-based and returns `{ data, error }` props.

Reply:
💯